src/d3a/models/strategy/pv.py

```diff
@@ -85,9 +85,10 @@ def _init_price_update(self, update_interval, initial_selling_rate, final_sellin
         validate_pv_device_price(fit_to_limit=fit_to_limit,
                                  energy_rate_decrease_per_update=energy_rate_decrease_per_update)
 
-        self.offer_update = UpdateFrequencyMixin(initial_selling_rate, final_selling_rate,
-                                                 fit_to_limit, energy_rate_decrease_per_update,
-                                                 update_interval)
+        self.offer_update = OffersUpdateFrequencyMixin(initial_selling_rate, final_selling_rate,
```

Comment:
Since we are in this part of the code, and the Mixin term is obviously wrong here (the PV/Load/StorageStrategy classes are not inheriting the Offers/BidsUpdateFrequencyMixin, thus this class is obviously not a Mixin but a normal class), how about renaming it to TemplateStrategyOfferUpdater and TemplateStrategyBidUpdater? @hannesdiedrich @fievelk feel free to pitch in if you disagree to the new naming.